ui/app/AppLayouts/Communities/popups/SharedAddressesPopup.qml

```diff
@@ -0,0 +1,50 @@
+import QtQuick 2.15
+
+import StatusQ.Popups.Dialog 0.1
+
+import AppLayouts.Communities.panels 1.0
+
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    property bool isEditMode
+
+    required property string communityName
```

Comment:
Maybe all of these properties can be an alias?!

Reply:
Can't be `required` then